settings.gradle

```diff
@@ -1,18 +1,20 @@
+pluginManagement {
+	repositories {
+		gradlePluginPortal()
+		maven { url 'https://repo.spring.io/plugins-release' }
+	}
+}
+
 plugins {
-	id 'com.gradle.enterprise' version '3.0'
+	id 'com.gradle.enterprise' version '3.1.1'
+	id "io.spring.gradle-enterprise-conventions" version "0.0.2"
 }
 
-if (System.getenv('GRADLE_ENTERPRISE_URL')) {
-	gradleEnterprise {
-		apply from: "$rootDir/gradle/build-scan-user-data.gradle"
-		buildScan {
-			captureTaskInputFiles = true
-			obfuscation {
-				ipAddresses { addresses -> addresses.collect { address -> '0.0.0.0'} }
-			}
-			publishOnFailure()
-			server = System.getenv('GRADLE_ENTERPRISE_URL')
-		}
+gradleEnterprise {
+	apply from: "$rootDir/gradle/build-scan-user-data.gradle"
+	buildScan {
+		captureTaskInputFiles = true
```

Comment:
You can remove this as the conventions plugin will take care of it.

Reply:
Thanks; my intention was to get your review before it was merged; sorry, @artembilan I should have made that clear.